samples/nrf9160/modem_shell/src/ppp/ppp_ctrl.c

```diff
@@ -36,9 +36,10 @@
 #include "ppp_ctrl.h"
 
 /* ppp globals: */
+bool ppp_up;
```

Comment:
Is this used outside `ppp_ctrl.c`? Doesn't look like it?

Reply:
Good point. Changed as static.